### DISCUSSION TITLE
Can get hypervisor virtualization on iPad Pro m1 with iOS15.6 soon?

### DISCUSSION THREAD

**Discussion by choibyeoli** [utmapp/UTM#4949] [UTM Help (iOS)] [answered]
Hello.
I want to use Windows 11 on my M1 iPad with full-speed UTM(hypervisor). but I'm on iOS15.6 and Trollstore not available. they said no more supported from now on.

I know that the macdirtycow exploit is now available and I seem that it's a way to use hypervisor.framework on UTM to -16.1.2 devices.

or it can't it doesn't have other way to use?

i just want to use hypervisor.framework

thank you for your hard work always 😉😉

**Comment by osy** [comment, marked as answer]
You are mistaken with what macdirtycow is. Rule of thumb, if you can’t run Cydia, you won’t be able to run UTM.HV.
